servicetalk-capacity-limiter-api/src/main/java/io/servicetalk/capacity/limiter/api/CapacityLimiter.java

```diff
@@ -101,7 +108,6 @@ interface Ticket {
          * Representation of the state of the {@link CapacityLimiter} when this {@link Ticket} was issued.
          * @return the {@link LimiterState state} of the limiter at the time this {@link Ticket} was issued.
          */
-        @Nullable
```

Comment:
There is one more implementation in `TrafficResilienceHttpServiceFilter` that still has this fields annotated as `@Nullable`. Consider removing that annotation.